src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Formatting;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Azure.WebJobs.Script.WebHost.Security;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Metrics
+{
+    public class LinuxContainerMetricsPublisher
+    {
+        public const string PublishMemoryActivityPath = "/memoryactivity";
+        public const string PublishFunctionActivityPath = "/functionactivity";
+
+        public const string ContainerNameHeader = "x-ms-functions-container-name";
+        public const string HostNameHeader = "x-ms-functions-host-name";
+        public const string SiteTokenHeader = "x-ms-site-restricted-token";
```

Comment:
Can you please define this one in ScriptConstants where we define some of the other headers? There are other places in the code where we use this header name - it'd be good to have them all use the same constant.